bobtemplates/odoo/hooks.py

```diff
@@ -164,6 +167,15 @@ def post_render_addon(configurator):
                      '/README.rst.oca')
         _delete_file(configurator, variables['addon.name'] +
                      '/static/description/icon.png.oca')
+    version = variables['addon.version']
+    if len(version) >= 2 and version[:2] == u'10':
```

Comment:
@adrienpeiffer `if len(version) >= 2` is superfluous.

To reliably compare version, you could use `pkg_resources.parse_version()`